frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/block-pool-update.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { checkErrors } from '@console/cypress-integration-tests/support';
+import {
+  createBlockPool,
+  deleteBlockPoolFromCli,
+  blockPoolFooter,
+  verifyBlockPoolJSON,
+} from '../views/block-pool';
+import { POOL_PROGRESS } from '../../src/constants/storage-pool-const';
+import { modal } from '../../../integration-tests-cypress/views/modal';
+
+describe('Test block pool update under OCS UI', () => {
+  before(() => {
+    cy.login();
+    cy.visit('/');
+    cy.install();
+  });
+
+  after(() => {
+    checkErrors();
+    cy.logout();
+  });
+
+  it('Check for a pool update', () => {
+    createBlockPool();
+    cy.log('Updating a newly created block pool');
+    cy.byLegacyTestID('kebab-button')
+      .first()
+      .click();
+    cy.byTestActionID('Edit Block Pool').click();
+
+    modal.modalTitleShouldContain('Edit Block Pool');
+    cy.byTestID('replica-dropdown').click();
+    cy.byLegacyTestID('replica-dropdown-item')
+      .contains('3-way Replication')
+      .click();
+    cy.byTestID('compression-checkbox').uncheck();
+
+    cy.log('Updating pool');
+    blockPoolFooter('update');
+
+    cy.log('Verify pool update');
+    verifyBlockPoolJSON(false, '3');
+
+    cy.log('Trying to update default pool');
```

Comment:
This should be a different test case.
`Test editing a default block pool is not allowed`